nautobot/core/tests/test_api.py

```diff
@@ -425,13 +425,13 @@ def test_serializer_class_required(self):
     def test_parse_success(self):
         status = extras_models.Status.objects.first()  # pylint: disable=redefined-outer-name
         tags = extras_models.Tag.objects.get_for_model(ipam_models.VLAN)
-        location = dcim_models.Location.objects.filter(parent__isnull=False).first()
+        vlan_group = ipam_models.VLANGroup.objects.filter(location__isnull=False).first()
 
         csv_data = "\n".join(
             [
                 # the "foobar" column is not understood by the serializer; per usual REST API behavior, it'll be skipped
-                "vid,name,foobar,description,status,tags,tenant,location__name,location__parent__name",
-                f'22,hello,huh?,"It, I say, is a living!",{status.name},"{tags.first().name},{tags.last().name}",,{location.name},{location.parent.name}',
+                "vid,name,foobar,description,status,tags,tenant,vlan_group__name,vlan_group__location__name",
+                f'22,hello,huh?,"It, I say, is a living!",{status.name},"{tags.first().name},{tags.last().name}",,{vlan_group.name},{vlan_group.location.name}',
```

Comment:
I believe so, since location is not a real model field anymore

Reply:
Since CSV import is REST serializer based now, I wonder if it'd be worth the effort to add `location` as an optional write-only field on the "new" serializer and make it be able to handle it?

Reply:
Added location field as an optional write-only field. But should we revert the test changes here?